bin/cli.js

```diff
@@ -94,9 +102,19 @@ async function bundle(main, command) {
   const bundler = new Bundler(main, command);
 
   if (command.name() === 'serve') {
-    const server = await bundler.serve(command.port || 1234);
```

Comment:
Default port is already handled:

https://github.com/parcel-bundler/parcel/blob/333c3aa5d20f98a5f3c52635751032d12854c13c/src/Bundler.js#L461-L465